src/ResourceManager/Profile/Commands.Profile.Test/EnvironmentCmdletTests.cs

```diff
@@ -72,10 +76,40 @@ public void AddsAzureEnvironment()
             Assert.Equal(env.Endpoints[AzureEnvironment.Endpoint.ManagementPortalUrl], cmdlet.ManagementPortalUrl);
             Assert.Equal(env.Endpoints[AzureEnvironment.Endpoint.Gallery], "http://galleryendpoint.com");
         }
-        
+
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
-        public void AddsEnvironmentMultipleTimes()
```

Comment:
I think we should have a recorded mocked test or a test marked 'liveonly' for this, but it is fine to add this later